pkg/cmdlinefilter/cmdlinefilter.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// The cmdlinefilter package provides filters to modify the bootloader kernel
```

Comment:
I think you could move this into the existing `pkg/cmdline`?

Reply:
Ok, shall I name the function RemoveFilter and UpdateFilter then ?

Reply:
Also I was wondering how to make it more generic so that the LinuxImage Load function could directly call it. and move the parameter handling completely to the cmds.

Reply:
Would an interface with a filter function that return the filtered cmdline do it ?
`Filter(cmdline string) string`
Then the update filter I'm doing could be a single struct parameter that would hold the 3 parameters.

What do you think ?

Reply:
Yes, sounds good

Reply:
done